tests/lax_numpy_test.py

```diff
@@ -1128,6 +1128,23 @@ def testUnique(self, shape, dtype, return_index, return_inverse, return_counts,
     jnp_fun = lambda x: jnp.unique(x, return_index, return_inverse, return_counts)
     self._CheckAgainstNumpy(onp_fun, jnp_fun, args_maker, check_dtypes=True)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_{}_{}".format(
+       jtu.format_shape_dtype_string(shape1, dtype1),
+       jtu.format_shape_dtype_string(shape2, dtype2)),
+       "shape1": shape1, "dtype1": dtype1, "shape2": shape2, "dtype2": dtype2,
+       "rng_factory": jtu.rand_default}
+      for dtype1 in [s for s in float_dtypes or int_dtypes if s != jnp.bfloat16]
+      for dtype2 in [s for s in float_dtypes or int_dtypes if s != jnp.bfloat16]
+      for shape1 in one_dim_array_shapes or [jtu.NUMPY_SCALAR_SHAPE]
+      for shape2 in one_dim_array_shapes or [jtu.NUMPY_SCALAR_SHAPE]))
+  def testUnion1d(self, shape1, dtype1, shape2, dtype2, rng_factory):
+    rng = rng_factory()
+    args_maker = lambda: [rng(shape1, dtype1), rng(shape2, dtype2)]
+    onp_fun = lambda ar1, ar2: onp.union1d(ar1, ar2)
+    jnp_fun = lambda ar1, ar2: jnp.union1d(ar1, ar2)
```

Comment:
These lambdas are a no-op, so you can drop them and use ``onp.union1d`` and ``jnp.union1d`` directly.